data/entities.yaml

```diff
@@ -216,6 +216,24 @@
   properties: [portable]
   capabilities: [float]
 
+- name: sand
+  display:
+    attr: copper
+    char: '░'
```

Comment:
Actually, I can not see the `░` character very well. Maybe its the dirt background and copper attribute?

Reply:
Oh, the name of the character is "light shade" and my font takes that literally:

![image](https://user-images.githubusercontent.com/44544735/172027803-3e56e418-b5d1-4c91-9f60-0ea0dbb316df.png)

Reply:
It's the same character used for dirt, grass, and stone, but should be the same color as copper.  I do want to make sure it's distinguishable in different terminals which don't all handle color the same...

Reply:
Hmm, in my terminal the copper and dirt colors are easily distinguishable.  But I tried it in gnome-terminal and the two colors are very very hard to tell apart.  Any suggestions for a different color and/or character to use?

Reply:
I would suggest `IceT` background and the `sand` attribute which I completely forgot about. :smile:

Reply:
Hah, didn't know we had a `sand` attribute!  I updated it, how does it look now?

Reply:
Looks good enough, in light colour schemes it very much looks like sand for me, in dark ones it's a bit muddy but still distinguishable. :+1: